packages/react-native-reanimated/src/platform-specific/RNRenderer.web.ts

```diff
@@ -1,3 +1,3 @@
-'use strict';
```

Comment:
May I restore `‘use strict’;` here if it does not break anything for you?

Reply:
should be fine to add back!

Reply:
Thanks!